Makefile

```diff
@@ -4,7 +4,7 @@
 
 PWD=$(shell pwd)
 
-BIN_DIR=/usr/local/bin
+BIN_DIR=$(HOME)/bin
```

Comment:
@inducer why did you make this change? This have broken the `sudo make install` command.

Reply:
Yikes, sorry. That was for my own use, it was never meant to make it into a commit.

Reply:
@inducer Could you make a new PR fixing that? Thank you!

Reply:
Fixed by 3b3772a